Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -553,94 +559,89 @@ def query_malops(total_result_limit=None, per_group_limit=None, template_context
+                    'IsIsolated': True
+                }
+            )
+        ]
     else:
         raise Exception('Failed to isolate machine.')
 
 
-def isolate_machine(machine):
-    pylum_id = get_pylum_id(machine)
+def isolate_machine(client: Client, machine: str) -> Any:
+    pylum_id = get_pylum_id(client, machine)
 
     cmd_url = '/rest/monitor/global/commands/isolate'
     json_body = {
         'pylumIds': [pylum_id]
 
     }
-    response = http_request('POST', cmd_url, json_body=json_body)
+    response = client.cybereason_api_call('POST', cmd_url, json_body=json_body)
 
     return response, pylum_id
 
 
-def unisolate_machine_command():
-    machine = demisto.getArg('machine')
-    response, pylum_id = unisolate_machine(machine)
-    result = response.get(pylum_id) if response else ''
+def unisolate_machine_command(client: Client, args: dict):
+    machine = str(args.get('machine'))
+    response, pylum_id = unisolate_machine(client, machine)
+    result = response.get(pylum_id)
     if result == 'Succeeded':
-        ec = {
-            'Cybereason(val.Machine && val.Machine === obj.Machine)': {
-                'Machine': machine,
-                'IsIsolated': False
-            },
-            'Endpoint(val.Hostname && val.Hostname === obj.Hostname)': {
-                'Hostname': machine
-            }
-        }
-        demisto.results({
-            'ContentsFormat': formats['json'],
-            'Type': entryTypes['note'],
-            'Contents': response,
-            'ReadableContentsFormat': formats['markdown'],
-            'HumanReadable': 'Machine was un-isolated successfully.',
-            'EntryContext': ec
-        })
+        return [
+            CommandResults(
+                readable_output='Machine was un-isolated successfully.',
+                outputs_prefix='Cybereason',
+                outputs_key_field='Machine',
+                outputs={
+                    'Machine': machine,
+                    'IsIsolated': False
+                }
+            )
+        ]
     else:
         raise Exception('Failed to un-isolate machine.')
 
 
-def unisolate_machine(machine):
-    pylum_id = get_pylum_id(machine)
+def unisolate_machine(client: Client, machine: str) -> Any:
+    pylum_id = get_pylum_id(client, machine)
     cmd_url = '/rest/monitor/global/commands/un-isolate'
     json_body = {
         'pylumIds': [pylum_id]
 
     }
-    response = http_request('POST', cmd_url, json_body=json_body)
+    response = client.cybereason_api_call('POST', cmd_url, json_body=json_body)
 
     return response, pylum_id
 
 
-def malop_processes_command():
-    malop_guids = demisto.getArg('malopGuids')
-    machine_name = demisto.getArg('machineName')
+def malop_processes_command(client: Client, args: dict):
+    malop_guids = args.get('malopGuids')
+    machine_name = str(args.get('machineName'))
+    date_time = str(args.get('dateTime'))
+
+    filter_input = []
+    if date_time:
```

Comment:
This is a logical issue-
if you don't get dateTime, the get('dateTime') will return None, which will be converted to 'None' due to the str() method.

So in that case, the `if` statement will actually ask `if "None"` and since this is not an empty string, it will return True when you expect False.
I would recommend adding a unitest for this use-case.

Reply:
> @suraj-metron, please let me know when you finished with my suggestion/ comments.

Sure @JasBeilin . I will definitely update you once we are done

Reply:
Thank you for poiting this out. We have fixed this issue as well as tested it against the Unit Test